distribution/docker/druid.sh

```diff
@@ -138,8 +138,8 @@ then
     setKey _common druid.zk.service.host "${ZOOKEEPER}"
 fi
 
-DRUID_SET_HOST=${DRUID_SET_HOST:-1}
-if [ "${DRUID_SET_HOST}" = "1" ]
+DRUID_SET_HOST_IP=${DRUID_SET_HOST_IP:-0}
```

Comment:
@georgew5656 could you please review this ?

Reply:
i'm a little confused why this is necessary, doesn't the current behavior also not set druid.host? since the default for DRUID_SET_HOST is -1 and the if statement for setting druid.host checks if that value is equal to 1

Reply:
@aho135 does this new default work okay with a raw docker run? i'm not exactly sure what 
InetAddress.getLocalHost().getCanonicalHostName() returns there? i think it might be the container-id which might not work that well since its not reachable

Reply:
@georgew5656 What did you have in mind as a raw docker run? The integration tests docker setup also uses InetAddress.getLocalHost().getCanonicalHostName() which just resolves to the name set in the [docker-compose](https://github.com/apache/druid/blob/master/integration-tests/docker/docker-compose.yml#L32)

Reply:
if the integration tests are passing then this seems fine to me